MIGRATION.md

```diff
@@ -265,17 +264,19 @@ import {
 </Button>;
 ```
 
+You can also make these changes manually.
+
 ## Icon
 
-Icon component in v4 has a simplified API. Simply import a required icon from `@contentful/f36-icons`.
+In v4, the Icon component has a simplified API. You can a required icon from `@contentful/f36-icons`.
```

Comment:
We missed the word "import" in the second phrase:
You can import a required icon from `@contentful/f36-icons`.